docs(jsx): add details to documentation (#909)
artalar/reatom: packages/jsx/README.md

)
```

> When there is a need to work with SVG as a string, you can use one of the following options.

Reviewer: ```suggestion
If you need to use SVG as a string, you can choose from these options:
```
WDYT?

- No DOM-less SSR (requires a DOM API implementation like `linkedom` to be provided)
- No keyed lists support

> - A component should have no more than one root element. If this interferes with the layout, you can wrap the parent elements in another element with the style `display: "contents"`:

Reviewer: Note for myself: We definitely need an eslint rule for that.
